game/scripts/vscripts/items/silver_staff.lua

```diff
@@ -140,11 +140,13 @@ function modifier_item_silver_staff_debuff:OnCreated()
       ability = ability,
     }
 
-    -- Don't do damage to bosses
-    if not parent:IsOAABoss() then
-      ApplyDamage(damage_table)
+    -- Do reduced damage to bosses
+    if parent:IsOAABoss() then
```

Comment:
bosses already take reduced percentage damage, that's the point of boss resistance... this is double scaling.